lib/6to5/transformation/transformers/playground-mallet-operator.js

```diff
@@ -0,0 +1,86 @@
+var t = require("../../types");
+
+var isMallet = function (node) {
+  var is = t.isAssignmentExpression(node) && node.operator === "||=";
+  if (is) {
+    var left = node.left;
+    if (!t.isMemberExpression(left) && !t.isIdentifier(left)) {
+      throw new Error("Expected type MemeberExpression or Identifier");
+    }
+    return true;
+  }
+};
+
+var getObjRef = function (node, nodes, file, scope) {
+  var obj = node.object;
+  if (t.isIdentifier(obj)) return obj;
```

Comment:
This is to avoid an unnecessary var declaration.

```javascript
// obj.x ||= 2
if (!obj.x) obj.x = 2


// As opposed to
var _obj = obj;
if (!_obj.x) _obj.x = 2
```